test/Microsoft.Dnx.Tooling.FunctionalTests/DnuRestoreTests.cs

```diff
@@ -1,193 +1,74 @@
-            {
-                var projectPath = testEnv.ProjectPath;
-                DirTree.CreateFromJson(projectStructure)
-                    .WithFileContents("project.json", projectJsonContent)
-                    .WithFileContents(scriptName, scriptContent)
-                    .WriteTo(projectPath);
-                FileOperationUtils.MarkExecutable(Path.Combine(projectPath, scriptName));
+            // Build a package for A
+            sdk.Dnu.Restore(solution.GetProject("A")).EnsureSuccess();
+            sdk.Dnu.Pack(
+                solution.GetProject("A").ProjectDirectory,
+                aOutputPath,
+                configuration: "package").EnsureSuccess();
+            sdk.Dnu.PackagesAdd(
+                Path.Combine(aOutputPath, "package", "A.1.0.0.nupkg"),
+                solution.LocalPackagesDir).EnsureSuccess();
 
-                string output;
-                string error;
-                var exitCode = DnuTestUtils.ExecDnu(
-                    runtimeHomePath,
-                    subcommand: "restore",
-                    arguments: null,
-                    stdOut: out output,
-                    stdErr: out error,
-                    environment: environment,
-                    workingDir: projectPath);
-
-                Assert.Equal(0, exitCode);
-                Assert.Empty(error);
-                Assert.Contains("Executing script 'prerestore' in project.json", output);
-                Assert.Contains("Executing script 'postrestore' in project.json", output);
-
-                var preContent = File.ReadAllText(Path.Combine(projectPath, "pre"));
-                Assert.Equal(expectedPreContent, preContent);
-                var postContent = File.ReadAllText(Path.Combine(projectPath, "post"));
-                Assert.Equal(expectedPostContent, postContent);
-            }
-        }
-
-        [Theory]
-        [MemberData(nameof(RuntimeComponents))]
-        public void DnuRestore_ReinstallsCorruptedPackage(string flavor, string os, string architecture)
-        {
-            var runtimeHomePath = _fixture.GetRuntimeHomeDir(flavor, os, architecture);
-            using (var tempDir = new DisposableDir())
-            {
-                var projectDir = Path.Combine(tempDir, "project");
-                var packagesDir = Path.Combine(tempDir, "packages");
-                var projectJson = Path.Combine(projectDir, Runtime.Project.ProjectFileName);
-
-                Directory.CreateDirectory(projectDir);
-                File.WriteAllText(projectJson, @"
-{
-  ""dependencies"": {
-    ""alpha"": ""0.1.0""
-  }
-}");
-                DnuTestUtils.ExecDnu(
-                    runtimeHomePath,
-                    subcommand: "restore",
-                    arguments: $"{projectDir} -s {_fixture.PackageSource} --packages {packagesDir}");
+            // Delete the project A
+            CommonTestUtils.TestUtils.DeleteFolder(solution.GetProject("A").ProjectDirectory);
```

Comment:
`DeleteFolder` is not in the new infrastructure, so the one from the old one is used. I'll probably add it in a future change.